scrc/preprocessors/extractors/citation_extractor.py

```diff
@@ -33,6 +33,9 @@ def __init__(self, config: dict):
             'processing_one': 'Extracting citations from',
             'no_functions': 'Not extracting citations.'
         }
+        
+    def get_coverage(self, engine: Engine, spider: str):
```

Comment:
Please remove the engine from the method heads.
Please log a short info message to the user everywhere it is not implemented.